### PR TITLE
Add babel and read .env file

### PR DESCRIPTION
**Pull request by KarolAltamirano**
- Add build process with babel
- Read token from .env file
- Add .editorconfig file
- Update README.md
- Create example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,26 @@ travis files for 50 different repos, you'll get the point :P
 
 **octopull** makes updating any set of files to any set of repositories a breeze.
 
+Currently works only with GitHub
 
 ## RDD - README driven development :P
 
+### Requirements
+- NodeJS v6.0 or newer
```
Comment (alvaropinot) [inline README.md:20]: let's add this also to the `package.json` :D

File: package.json [hunk 2]
```diff
@@ -2,26 +2,33 @@
...
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/alvaropinot/octopull.git"
-  },
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "repository": "alvaropinot/octopull",
+  "author": "Alvaro Pinot <alvaro@neatnait.com> (http://github.com/alvaropinot)",
```
Comment (alvaropinot) [inline package.json:8]: add your self as a contributor :P

File: package.json [hunk 3]
```diff
@@ -2,26 +2,33 @@
...
     "git"
   ],
-  "author": "Alvaro Pinot <alvaro@neatnait.com> (http://github.com/alvaropinot)",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/alvaropinot/octopull/issues"
   },
-  "homepage": "https://github.com/alvaropinot/octopull#readme",
+  "scripts": {
+    "clean": "rimraf lib",
```
Comment (alvaropinot) [inline package.json:21]: ❤️

File: package.json [hunk 4]
```diff
@@ -2,26 +2,33 @@
...
+    "dev": "npm run clean && babel src -w -s -d lib",
+    "build": "npm run clean && babel src -d lib",
+    "prepublish": "npm run build"
+  },
```
Comment (alvaropinot) [inline package.json]: out of scope of the PR, but we may add lint script here with `standard` or something. What do you think?

**Review by alvaropinot — APPROVED** [review]
LGTM thank you @KarolAltamirano !!!

I'll test the changes locally and merge the PR ASAP :D

File: README.md (hunk 1 above)
Comment (KarolAltamirano) [reply README.md:20]: Added :)

File: package.json (hunk 2 above)
Comment (KarolAltamirano) [reply package.json:8]: Hahaha, added 👍

File: package.json (hunk 4 above)
Comment (KarolAltamirano) [reply package.json]: Great idea, I've added eslint with some initial config. Let's take the config as a starting point and we can update it how we like :)

**Comment by KarolAltamirano** [comment]
I've also added CHANGELOG.md and LICENSE.md files :)

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (alvaropinot) [inline .eslintrc]: Maybe I would prefer https://github.com/feross/standard as there is no possible discussions because it is as it is.

**Review by alvaropinot — APPROVED** [review]
❤️ 

Implements #5

**Comment by KarolAltamirano** [comment]
PR updated to use 'standard' (https://github.com/feross/standard)

**Review by alvaropinot — APPROVED** [review]

**Pull request merged**